static/app/views/projectDetail/projectScoreCards/projectStabilityScoreCard.spec.tsx

```diff
@@ -0,0 +1,151 @@
+import {Organization} from 'sentry-fixture/organization';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {SessionFieldWithOperation} from 'sentry/types';
+import ProjectStabilityScoreCard from 'sentry/views/projectDetail/projectScoreCards/projectStabilityScoreCard';
+
+describe('ProjectDetail > ProjectStability', function () {
+  let endpointMock: jest.Mock;
```

Comment:
not really shared between tests, consider using a constant

Reply:
Will remove and fix were I copied it from 👍